wagtail/admin/templates/wagtailadmin/shared/breadcrumb-next.html

```diff
@@ -0,0 +1,67 @@
+{% load i18n wagtailadmin_tags %}
+
+{% with breadcrumb_link_classes='w-flex w-justify-start w-items-center w-text-grey-600 w-no-underline w-outline-offset-inside hover:w-underline hover:w-text-primary' breadcrumb_list_classes='w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0' icon_classes='w-w-4 w-h-4 w-mr-3' %}
+    {# Breadcrumbs are visible on mobile by default but hidden on desktop #}
+    <button data-toggle-breadcrumbs  {# handled in page-editor.js #}
```

Comment:
this needs `type="button"` as the default is `submit` which could lead to unexpected behaviours. ref - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button